docs/next/guides/getting-started/setting-options.md

```diff
@@ -14,319 +14,219 @@ tags:
 
 ## Overview
 
-## Available options
-
-Opisać core i plugins i wrzucić linki do reference do obu
-
-Najlepiej jakby to było dostępne też dla wrapperów
-
-## The cascading configuration
+Handsontable uses cascading configuration, a fast way to provide configuration options for the entire table, including its columns and particular cells. To show you how it works, we use the [readOnly](@/api/metaSchema.md#readOnly) and [className](@/api/metaSchema.md#className) options, which we will play in all the examples below.
 
 ### Entire grid
 
-Contructor
-
-### Single cell
-
-Address by row / col order; A1, B2; ID column and row
+By settings the [readOnly](@/api/metaSchema.md#readOnly) option, we tell Handsontable to propagate the option through the column settings down to the cells. Hence, all cells have a read-only state.
```

Comment:
By settings > By setting

we tell Handsontable to propagate > we propagate

Hence > As a result